docker/docker_image_src.go

```diff
@@ -399,7 +405,22 @@ func (s *dockerImageSource) GetBlobAt(ctx context.Context, info types.BlobInfo,
 	case http.StatusPartialContent:
 		mediaType, params, err := mime.ParseMediaType(res.Header.Get("Content-Type"))
 		if err != nil {
-			return nil, nil, err
+			if err == mime.ErrInvalidMediaParameter {
+				// CloudFront returns an invalid MIME type, that contains an unquoted ":", let's
```

Comment:
I don’t quite see how this comment `:` relates to the code, but, I guess we’ll see from the test case…

Do you mean that it is invalid to have a `;` in the _boundary_? There is no `:` in `multipart/byteranges`.

Reply:
Ah, the PR description does say “boundary”.